Content.Server/Light/EntitySystems/ExpendableLightSystem.cs

```diff
@@ -200,5 +202,11 @@ private void AddIgniteVerb(Entity<ExpendableLightComponent> ent, ref GetVerbsEve
             };
             args.Verbs.Add(verb);
         }
+
+        private void IsHotEvent(Entity<ExpendableLightComponent> ent, ref IsHotEvent args)
+        {
+            if (HasComp<IgnitionSourceComponent>(ent))
+                args.IsHot = ent.Comp.Activated;
+        }
     }
```

Comment:
1. This might set IsHot to false if there's multiple sources on this.
2. Checking a bool property is significantly faster than hascomp so do that first.

Reply:
doesnt ignition source already handle IsHotEvent anyway

Reply:
> doesn't the ignition source already handle IsHotEvent anyway

It's reading the event and setting ignited from it. It would also be inconsistent with how everywhere else does it.